compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt

```diff
@@ -816,10 +849,9 @@ private class AccessibilityContainer(
     }
 }
 
-private sealed interface NodesSyncResult {
-    object NoChanges : NodesSyncResult
-    data class Success(val newElementToFocus: Any?) : NodesSyncResult
-}
+private class NodesSyncResult(
+    val newElementToFocus: Any?
```

Comment:
`AccessibilityElement?` can be used here

Reply:
Right now - yes, but in next milestone we will also need to support interop views, so this will turn into some sealed interface or Any?